src/main/java/com/pubnub/api/builder/PubNubErrorBuilder.java

```diff
@@ -376,6 +376,11 @@ public final class PubNubErrorBuilder {
      */
     public static final int PNERR_USERID_CAN_NOT_BE_DIFFERENT = 173;
 
+    /**
+     * Used when crypto is configured but the message was not encrypted.
+     */
+    public static final int PNERR_CRYPTO_IS_CONFIGURED_BUT_MESSAGE_IS_NOT_ENCRYPTED = 177;
```

Comment:
Why not to assign 174 instead of 177 ?
174 is next in sequence since receding error is 173
PNERR_USERID_CAN_NOT_BE_DIFFERENT = 173

Reply:
are our error codes different between SDKs? 😨 I can change it, I wanted to be consistent with Kotlin SDK

Reply:
This is why we encourage matching errors using associated constant names :)

Reply:
done